### DISCUSSION TITLE
Case-insensitive sort

### DISCUSSION THREAD

**Discussion by mlbiche** [directus/directus#17339] [Feature Requests]
It seems that the sort feature is case-sensitive. When sorting by a text field, it results in :

```
A
B
a
b
```

It would be nice to be able to apply case-insensitive sort, for the following expected result :

```
A
a
B
b
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
